tests/scene/test_sky.h

```diff
@@ -136,6 +136,89 @@ TEST_CASE("[SceneTree][Sky] Material setter and getter") {
 	memdelete(test_sky);
 }
 
+TEST_CASE("[SceneTree][Sky] Invalid radiance size handling") {
+	Sky *test_sky = memnew(Sky);
```

Comment:
Should `Ref<Sky>` be used instead? This way, you don't need to `memdelete()` it at the end.

Reply:
I followed the format of the original tests in `test_sky.h`, which used `memnew()` and `memdelete()`. Do you think it's okay if I change all of them?